src/App.css

```diff
@@ -1,6 +1,10 @@
+* {
+  scroll-behavior: smooth;
+}
+
 .App {
   min-height: 100vh;
-  background-color: grey;
+  background-color: #282c34;
```

Comment:
I'd use Css variables for colors:
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

Reply:
I implement CSS variables for main colors.